src/@next/components/atoms/ButtonLink/__snapshots__/stories.storyshot

```diff
@@ -33,6 +33,12 @@ exports[`Storyshots @components/atoms/ButtonLink Base 1`] = `
   -webkit-text-decoration: underline;
   text-decoration: underline;
   padding: 0;
+  -webkit-transition: color 0.3s ease;
```

Comment:
no need to use webkit and other custom browser-engine specific CSS styles, styled-components auto-applies them in runtime

Reply:
there is no -webkit prefix added in styles, it's only in snapshots

Reply:
🤦‍♂ right